Package.swift

```diff
@@ -1,12 +1,13 @@
-// swift-tools-version:5.5
+// swift-tools-version:5.10
```

Comment:
I thin we can avoid bumping the tools by using custom platform, that may be an issue to some deployments

```swift
.custom("visionos", versionString: "1.0")
```

also .visionos was introduced in 5.9

Reply:
Thats a very good point.

Let me rollback first and update second to:
`.custom("xros", versionString: "1.0")`